src/cli/commands/install.js

```diff
@@ -660,7 +660,7 @@ export class Install {
     }
 
     // build lockfile location
-    const loc = path.join(this.config.cwd, constants.LOCKFILE_FILENAME);
+    const loc = path.join(this.config.worktreeFolder || this.config.cwd, constants.LOCKFILE_FILENAME);
```

Comment:
The question is - someone adds a new command, say `amazing-command`, do they need to pass `this.config.worktreeFolder || this.config.cwd` when reading lockfile?

It looks like we leak workspaces implementation detail into every command, seems like a red flag and besides, we have `config.worktreeFolder ||` repeated 13 times already. It definitely asks for encapsulation